templates/storypages/index.html

```diff
@@ -19,20 +19,40 @@
     <!-- The slides container -->
     <section class="slide-section">
       <article class="slide" id="title-slide">
-        <h1>Story Map title</h1>
-        <p>Lorem ipsum dolor sit amet, consectetur adipiscing elit. Suspendisse molestie hendrerit quam, efficitur vestibulum justo vehicula a.</p>
-        <p>Nullam et metus eu eros facilisis commodo. Vivamus elementum nulla nisi, eget fermentum mauris placerat sed.</p>
+        <h1>History of Philadelphia</h1>
+        <img class = "img-eye" src="https://assets.visitphilly.com/wp-content/uploads/2018/06/philadelphia-world-heritage-city.png" width="200"></img>
```

Comment:
here (and everywhere you create an  element of class "img-eye": lines 23, 40, 47, 54), when running AxeDevTools in my developer window I see that you have an accessibility issue. It suggests to add an `alt` argument to add alternative text that could be read out load if someone could not see the image

```suggestion
        <img class = "img-eye" alt = "A seal of Philadelphia showing an image of city hall and reading Philadelphia world heritage city" src="https://assets.visitphilly.com/wp-content/uploads/2018/06/philadelphia-world-heritage-city.png""></img>
```

for more information, see https://dequeuniversity.com/rules/axe/4.8/image-alt?application=AxeChrome